commands/sandbox.go

```diff
@@ -347,18 +356,17 @@ func RunSandboxExecStreaming(command string, c *CmdConfig, booleanFlags []string
 // We don't expect both Table and Entity to be present and have no
 // special handling for that.
 func (c *CmdConfig) PrintSandboxTextOutput(output do.SandboxOutput) error {
+	var err error
 	if len(output.Formatted) > 0 {
-		fmt.Fprintf(c.Out, strings.Join(output.Formatted, "\n"))
+		_, err = fmt.Fprintln(c.Out, strings.Join(output.Formatted, "\n"))
 	} else if len(output.Captured) > 0 {
-		fmt.Fprintf(c.Out, strings.Join(output.Captured, "\n"))
+		_, err = fmt.Fprintln(c.Out, strings.Join(output.Captured, "\n"))
 	} else if len(output.Table) > 0 {
-		fmt.Fprintf(c.Out, genericJSON(output.Table))
+		_, err = fmt.Fprintln(c.Out, genericJSON(output.Table))
 	} else if output.Entity != nil {
-		fmt.Fprintf(c.Out, genericJSON(output.Entity))
+		_, err = fmt.Fprintln(c.Out, genericJSON(output.Entity))
 	} // else no output (unusual but not impossible)
 
-	_, err := fmt.Fprintln(c.Out)
```

Comment:
This change is unrelated to the substance of the PR but one user complained that there is a spurious newline for commands that produce no output.   I suppose it's arguable whether this matters and/or which behavior is better but I thought to remove that one possible irritant.   The original `nim` behavior was no newline.